.gitlab-ci.yml

```diff
@@ -63,11 +61,26 @@ build:testing:
     paths:
       - testingImage.tar
 
+build:qa:
+  stage: build
+  tags:
+    - mender-qa-slave-highcpu
+  services:
+    - docker:dind
+  script:
+    - docker build -t $DOCKER_REPOSITORY:qa-acceptance-testing -f qa-acceptance-testing/Dockerfile qa-acceptance-testing
+    - docker save $DOCKER_REPOSITORY:qa-acceptance-testing > qaTestingImage.tar
```

Comment:
Suggest renaming from `qa-acceptance-testing` to `mender-qa-acceptance-testing` so that the association is explicit.

Reply:
Actually, this is still confusing to me. This image is for client acceptance tests, so I would be even more explicit with for example `mender-qa-client-acceptance-testing`.

Reply:
considering the confusion I'll add a `README` with a mapping of the tags to the jobs they are used in... just need to figure out the `raspbian-latest` usage 😄
regarding the naming, I thought the mender context should be clear given that this is a `mender-test-container` image... 🤷‍♂️
and for the client part at least https://github.com/mendersoftware/mender-qa/blob/master/.gitlab-ci.yml#L538 disagrees... there I'm wondering what these dependencies should be used for in the building of the backend services, however that is part of another PR in the future...

Reply:
now I changed it to `mender-client-acceptance-testing` - afaik the only acceptance testing we do is in the qa repo right? I hope that's ok...
the small readme should hopefully give enough info, unfortunately I haven't found an example of the `raspbian_latest` image in use - if you can point me to one I'll add this in too 👌

Reply:
Thanks for the README file. See my comment there for where the `raspbian_latest` is used (this we could rename easily later on).

I agree that we don't need `mender-qa` in the names (theoretically speaking, I could pull these images locally to run tests) and I also agree that the repo name `mender-test-containers` is enough info about testing. So generally speaking, I like your new suggested names!